cmake/AddFdbTest.cmake

```diff
@@ -37,6 +37,9 @@ function(configure_testing)
 endfunction()
 
 function(verify_testing)
+  if(NOT ENABLE_SIMULATION_TESTS)
+    return()
+  endif()
```

Comment:
Same as in the 7.0 cherry-pick: please remove this code block.

Reply:
Done.